### PR TITLE
Create ABNT NBR 6023:2018. UNESP – Faculdade de Engenharia de Guarati…

### PR DESCRIPTION
**Pull request by suelencfmorais**
Hello ,
I request the inclusion of my Institution's style in the community repository.
Thank you.

### CODE REVIEW

**Comment by adam3smith** [comment]
THe remaining errors are validation errors: https://validator.citationstyles.org/?url=https%3A%2F%2Fraw.githubusercontent.com%2Fcitation-style-language%2Fstyles%2F6ad148ab5a131a34bac32cca5029fec8f2809035%2Funiversidade-estadual-paulista-faculdade-de-engenharia-de-guaratingueta-abnt.csl&version=1.0.1
You'll need to fix them -- see above for how to update your pull request

**Comment by suelencfmorais** [comment]
I realized that my Instituiton's style isn't available. What's the next step needed for the style availability in repository, which can be found in Zotero and Mendeley?

**Comment by adam3smith** [comment]
I just noticed -- there's no link to documentation for the style -- are the guidelines available online somewhere? Where?

**Comment by suelencfmorais** [comment]
Hello, we changed the style and put the address where the style is hosted. We would like to know if this will integrate the style into the international repository for zotero and mendeley.

File: universidade-estadual-paulista-faculdade-de-engenharia-de-guaratingueta-abnt.csl [hunk 1]
```diff
@@ -0,0 +1,1033 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="never" default-locale="pt-BR">
+  <info>
+    <title>Universidade Estadual Paulista - Faculdade de Engenharia de Guaratingueta- ABNT (Português - Brasil)</title>
```
Comment (adam3smith) [inline universidade-estadual-paulista-faculdade-de-engenharia-de-guaratingueta-abnt.csl:4]:
```suggestion
    <title>Universidade Estadual Paulista - Faculdade de Engenharia de Guaratingueta - ABNT (Português - Brasil)</title>
```

**Review by adam3smith — CHANGES_REQUESTED** [review]

**Comment by adam3smith** [comment]
Sorry for the delay!

**Comment by suelencfmorais** [comment]
Hello, I can't find in Mendeley the style that I submited, I just found in Zotero. Someone coul'd help with this? tks.

**Comment by adam3smith** [comment]
That's a question for Mendeley; we have no insight into when/how they update styles.

**Comment by imsys** [comment]
@suelencfmorais, I'm using Zotero and I found the style you created extremely helpful!

I'm just checking the [NBR 6023:2018 documentation](http://plone.ufpb.br/secretariado/contents/documentos/2020ABNT60232018VersoCorrigida.pdf) and it says it can show the ISBN and DOI just before the URLs, so I would suggest to add the following inside the "access" macro, or in your preferred way.

```xml
  <macro name="access">
    <text variable="ISBN" prefix="ISBN " suffix=". "/>
    <text variable="DOI" prefix="DOI " suffix=". "/>
    <choose>
```
I also would like to know if we could create a numbered version, like: ²³, we could use `nature.csl` as a basis for working on that.

**Pull request merged**